e2e/cypress/integration/messaging/direct_messages_spec.js

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+import users from '../../fixtures/users.json';
+
+const userActive = users['user-1'];
+const userInactive = users['user-2'];
+
+describe('Messaging', () => {
+    afterEach(() => {
+        // # Activate "user-2"
+        cy.apiLogin('sysadmin');
+        cy.apiGetUserByEmail(userInactive.email).then((res) => {
+            const user = res.body;
+            cy.apiActivateUser(user.id, true);
+        });
+    });
```

Comment:
The first test failed either on retry or if other specs made previous conversation with "user-2". Instead of "user-2", I suggests to create "use-and-forget" new user as a test user. With that, you don't need to reactivate the user in afterEach hook. Something like,
```javascript
let testUser;
beforeEach(() => {
    cy.apiLogin('sysadmin');
    cy.apiCreateNewUser().then((user) => {
        testUser = user;
    });
});
```

Reply:
It's a good suggestion. I used Cypress aliasing. I'm not sure if this is better by any means, I just wanted to see if it works :) let me know if you want me to change it to variable assignment

Reply:
I'm good with it.  See comment on usage.